src/OpenTelemetry/Metrics/Reader/PeriodicExportingMetricReader.cs

```diff
@@ -33,8 +32,32 @@ public PeriodicExportingMetricReader(
         BaseExporter<Metric> exporter,
         int exportIntervalMilliseconds = DefaultExportIntervalMilliseconds,
         int exportTimeoutMilliseconds = DefaultExportTimeoutMilliseconds)
+        : this(exporter, new PeriodicExportingMetricReaderOptions
+        {
+            ExportIntervalMilliseconds = exportIntervalMilliseconds,
+            ExportTimeoutMilliseconds = exportTimeoutMilliseconds,
+            UseThreads = true,
+        })
+    {
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="PeriodicExportingMetricReader"/> class.
+    /// </summary>
+    /// <param name="exporter">Exporter instance to export Metrics to.</param>
+    /// <param name="options">Configuration options for the periodic exporting metric reader.</param>
+    public PeriodicExportingMetricReader(
+        BaseExporter<Metric> exporter,
+        PeriodicExportingMetricReaderOptions options)
         : base(exporter)
     {
+        Guard.ThrowIfNull(options);
+
+#pragma warning disable CA1062 // Validate arguments of public methods - needed for netstandard2.1
+        var exportIntervalMilliseconds = options!.ExportIntervalMilliseconds ?? DefaultExportIntervalMilliseconds;
+#pragma warning restore CA1062 // Validate arguments of public methods - needed for netstandard2.1
```

Comment:
Would using `?` instead of `!` make it happy without needing the `#pragma`?

Reply:
It does, though it needs to be specified in all uses of `options`. I've mainly used this because it's the same pattern used all around the library, but null propagation is also fine :) Will adjust.